src/main/java/com/dreammaster/mantle/MantleBookLoader.java

```diff
@@ -0,0 +1,79 @@
+package com.dreammaster.mantle;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.StatCollector;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+
+import gregtech.GT_Mod;
+import gregtech.api.interfaces.internal.IGT_Mod;
+import mantle.books.BookData;
+
+final class MantleBookLoader implements BookLoader {
+
+    static final BookDataReader BOOK_DATA_READER = new BookDataReader();
+    static final BookDataStoreProxy BOOK_DATA_STORE_PROXY = BookDataStoreProxy.getInstance();
```

Comment:
These fields should be static, shouldn't it?

Reply:
They are static, but they could be private.